From 7de22fc7c2726b2358225a7f52327b4902c24b5a Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Fri, 17 Jan 2025 23:23:10 +0100
Subject: [PATCH 2/4] mediatek: filogic: Cudy WR3000H: Fix SUPPORTED_DEVICES

Fix the name used for the transition image from Cudy.
This should make it possible to use the cudy transition image.

Fixes: 9d66b8b312fb ("mediatek: filogic: Add support for cudy wr3000h")
Link: https://github.com/openwrt/openwrt/pull/17652
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
(cherry picked from commit 0f8a5a5fd6929a3795a67ad4f98a7bc7358d22d9)
---
 target/linux/mediatek/image/filogic.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/mediatek/image/filogic.mk b/target/linux/mediatek/image/filogic.mk
index 76cca201ed..c1e86f5d5d 100644
--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -764,7 +764,7 @@ define Device/cudy_wr3000h-v1
   DEVICE_VARIANT := v1
   DEVICE_DTS := mt7981b-cudy-wr3000h-v1
   DEVICE_DTS_DIR := ../dts
-  SUPPORTED_DEVICES += R59
+  SUPPORTED_DEVICES += R63
   UBINIZE_OPTS := -E 5
   BLOCKSIZE := 128k
   PAGESIZE := 2048
-- 
2.43.0

